libbeat/outputs/elasticsearch/docs/elasticsearch.asciidoc

```diff
@@ -759,6 +759,68 @@ output.elasticsearch:
     index: "my-dead-letter-index"
 ------------------------------------------------------------------------------
 
+===== `preset`
+
+The performance preset to apply to the output configuration.
+
+["source","yaml"]
+------------------------------------------------------------------------------
+output.elasticsearch:
+  hosts: ["http://localhost:9200"]
+  preset: balanced
+------------------------------------------------------------------------------
+
+Performance presets apply a set of configuration overrides based on a desired performance goal. If set, a performance preset will override other configuration flags to match the recommended settings for that preset. Valid options are `balanced` (good starting point for general efficiency), `throughput` (good for high data volumes, may increase cpu and memory requirements), `scale` (reduces ambient resource use in large low-throughput deployments), `latency` (minimize the time for fresh data to become visible in Elasticsearch), and `custom` (apply user configuration directly with no overrides). The default if unspecified is `custom`.
+
+Presets represent current recommendations based on the intended goal; their internal effect may change between versions. Currently the presets have the following effects:
+
+[cols="2,1,1,1,1"]
+|===
+|preset |balanced |throughput |scale |latency
+
+|`bulk_max_size`
```

Comment:
Would it be possible to link these settings to their respective documentation areas?  For example, this one would link to https://www.elastic.co/guide/en/beats/filebeat/current/elasticsearch-output.html#_bulk_max_size.